src/sentry/api/endpoints/organization_events.py

```diff
@@ -196,12 +204,64 @@ def get(self, request, organization):
             }, status=400)
 
         try:
-            tag_key = tagstore.get_group_tag_keys_and_top_values(
+            tag_keys = tagstore.get_group_tag_keys_and_top_values(
                 project_ids, None, environment_ids, keys=lookup_keys, get_excluded_tags=True, **snuba_args)
         except tagstore.TagKeyNotFound:
             raise ResourceDoesNotExist
 
-        return Response(serialize(tag_key, request.user))
+        if non_tag_lookup_keys:
+            tag_keys.update(self.handle_non_tags_keys(non_tag_lookup_keys, snuba_args))
+
+        return Response(serialize(tag_keys, request.user))
+
+    def handle_non_tags_keys(self, keys, snuba_args):
+        result = set([])
+        for key in keys:
+            data = self.__query_non_tag_data('project_id', snuba_args)
+            if key == 'project':
+                projects = Project.objects.filter(id__in=snuba_args['filter_keys']['project_id'])
+                for project_data in data:
+                    project = projects.filter(id=project_data['project_id'])[0]
+                    project_data['key'] = 'project'
+                    project_data['value'] = project.slug
+            result.add(self.__create_tag_key_tag_value_objects('project', data))
+        return result
+
+    def __query_non_tag_data(self, key, snuba_args):
```

Comment:
What does two `_` do?

Reply:
I think in python it makes it harder to call from outside the class. I used to use them for "private" methods. I can use use one though. That looks like what we do.